src/content/extensions/multipleroutes/src/eventPage.js

```diff
@@ -1 +1,74 @@
-chrome.privacy.network.webRTCMultipleRoutesEnabled.set({'value': false});
+'use strict';
+
+// This file sets the policy when the extension is installed and registered for
+// chrome.runtime.onInstalled event to convert the booleans in pre-M48 version
+// to IPHandlingPolicy when chrome is upgraded to M48.
+
+// If this is installed in a pre-M48 version of Chrome, the only thing to do
+// here is to disable MultipleRoute.
+var pn = chrome.privacy.network;
+if (pn.webRTCIPHandlingPolicy === undefined) {
+  pn.webRTCMultipleRoutesEnabled.set({
+    value: false
+  });
+}
+
+// This function resets the 2 booleans to default values so we can ignore them
+// as if they were not set in future chrome.runtime.onInstalled event. This is
+// to avoid repeated conversions and overwrite the current setting.
+function resetOldBooleans(callback) {
+  pn.webRTCNonProxiedUdpEnabled.set({
+    value: true
+  }, function() {
+    pn.webRTCMultipleRoutesEnabled.set({
+      value: true
+    }, function() {
+      if (callback) {
+        callback('Successfully reset the boolean');
```

Comment:
Why are we hardcoding a message in this function?  Why not just fire the callback and let the callback handle the message?